src/main/storm/mesos/MesosNimbus.java

```diff
@@ -890,7 +890,7 @@ private Credential getCredential(FrameworkInfo.Builder finfo) {
       if (StringUtils.isNotEmpty(secretFilename)) {
         try {
           // The secret cannot have a NewLine after it
-          credentialBuilder.setSecret(ByteString.readFrom(new FileInputStream(secretFilename)));
```

Comment:
@DarinJ is changing this in a different way (using reflection) to support both mesos-0.26.0 and earlier versions.